routes/users.py

```diff
@@ -48,3 +48,17 @@ async def delete_user(identifier: models.UserIdentifier):
 async def get_user(identifier: models.UserIdentifier):
     user_data = await utils.get_user_info_by_identifier(identifier)
     return models.UserInfoQueryResponse(**user_data.dict())
+
+
+# this implementation doesn't use models.ValidateUserResponse. Should I implement this?
```

Comment:
Yep. Whatever data the method below returns will be cast into a `ValidateLoginResponse`, so if the data you're returning _can't_ be cast, you'll get an error. 

The easiest way to do this is to make sure that the `utils` method that you're calling in the function below returns a `ValidateLoginResponse` model.